index/slashing/slashing_test.go

```diff
@@ -11,12 +11,14 @@ import (
 )
 
 func TestMain(m *testing.M) {
-	util.AvgBlockTime = time.Millisecond * 10
+	util.AvgBlockTime = time.Millisecond * 100
 	logging.SetAllLoggers(logging.LevelError)
 	os.Exit(m.Run())
 }
 
 func TestFreshIndex(t *testing.T) {
+	// Skipped until #235 lands.
+	t.SkipNow()
```

Comment:
Is green locally, but is flaky on CI. 
There's some pending work to do about the slashing index anyway, so this test doesn't make much sense. In fact, it never made much sense since there isn't a good way to force slashing.